src/github.com/matrix-org/dendrite/roomserver/input/state.go

```diff
@@ -25,6 +25,12 @@ func calculateAndStoreState(
 		return 0, err
 	}
 
+	return calculateAndStoreStateAfterEvents(db, roomNID, prevStates)
```

Comment:
I don't understand why the return value for `calculateAndStoreStateBeforeEvent` is the same as `calculateAndStoreStateAfterEvents`. Please explain.